pkg/util/template/translation.go

```diff
@@ -34,6 +34,9 @@ var appSpecificKeysRegex = []*regexp.Regexp{
 	regexp.MustCompile(`^email\..+\.sender$`),
 	regexp.MustCompile(`^email\..+\.reply-to$`),
 	regexp.MustCompile(`^sms\..+\.sender$`),
+	regexp.MustCompile(`^terms-of-service-link$`),
+	regexp.MustCompile(`^customer-support-link$`),
+	regexp.MustCompile(`^privacy-policy-link$`),
```

Comment:
Add a test case in pkg/util/template/translation_test.go. It simply be the case mentioned in the issue. The test can also serve as a documentation of the intended behavior.

Reply:
Sorry I didn't make myself very clear. I actually mean adding a test as a sibling to https://github.com/authgear/authgear-server/blob/main/pkg/util/template/translation_test.go#L227-L250 like this

```
Convey("it should use the privacy-policy-link defined in the default language", func() {... })
Convey("it should use the customer-support-link defined in the default language", func() { ... })
Convey("it should use the terms-of-service-link defined in the default language", func() { ... })
```